### DISCUSSION TITLE
Directives

### DISCUSSION THREAD

**Discussion by hogmuzzle** [smallrye/smallrye-graphql#1008] [Q&A]
Can you add an example of custom directives usage. I tried to add directives but they are not printed in schema. Is there any working example?

**Comment by phillip-kruger** [comment]
@t1 might have an example

**Reply by t1** [reply]
I only use custom directives in the Federation context, so they are in the main branch.

My first guess is that you need to enable the printing of directives with the `smallrye.graphql.schema.includeDirectives` config.

**Reply by hogmuzzle** [reply]
Yes I have found this config but it is not reachable in quarkus. I should open a discussion there. Thank you.

**Comment by phillip-kruger** [comment]
Directives is not yet in Quarkus. It's on the TODO list ...
